app/components/Map.js

```diff
@@ -12,9 +12,11 @@ export default class Map extends Component {
     this.state = {
       position: null,
     };
-    this.testRef = new Firebase("https://interruptedlobster.firebaseio.com/pins");
```

Comment:
Removed this.testRef here, but still reference it on line 81, causing a crash when I try to place a pin